qutebrowser/browser/pdfjs.py

```diff
@@ -30,11 +30,11 @@
 
 _SYSTEM_PATHS = [
     # Debian pdf.js-common
-    # Arch Linux pdfjs (AUR)
+    # Arch Linux pdfjs
     '/usr/share/pdf.js/',
     # Flatpak (Flathub)
     '/app/share/pdf.js/',
-    # Arch Linux pdf.js (AUR)
+    # Arch Linux pdf.js
```

Comment:
This doesn't actually exist anymore, but the path might still be useful to check - maybe add a `(defunct)` comment rather than removing `(AUR)` here?

Reply:
I've removed `AUR` because the path `/usr/share/pdf.js/` is the folder where I do have something in it. If I put `(defunct)` it looks like the path is not valid anymore, doesn't it?

Reply:
But this comment is for `/usr/share/javascript/pdf.js/`, not `/usr/share/pdf.js/`.